src/vstart.sh

```diff
@@ -278,10 +282,14 @@ shift
 done
 
 if [ "$overwrite_conf" -eq 0 ]; then
-  CEPH_NUM_MON=`awk -F= '/CEPH_NUM_MON/{print $2}' $conf_fn`
-  CEPH_NUM_OSD=`awk -F= '/CEPH_NUM_OSD/{print $2}' $conf_fn`
-  CEPH_NUM_MDS=`awk -F= '/CEPH_NUM_MDS/{print $2}' $conf_fn`
-  CEPH_NUM_RGW=`awk -F= '/CEPH_NUM_RGW/{print $2}' $conf_fn`
+    MON=`awk -F= '/CEPH_NUM_MON/{print $2}' $conf_fn`
```

Comment:
Can you just do something like: CEPH_NUM_MON=`awk -F= '/CEPH_NUM_MON/{print $2}' $conf_fn` before line 79?  If the awk fails, it will fall back to the defaults.